rpc_jobs/release.yml

```diff
@@ -0,0 +1,90 @@
+- job:
+    name: RE-Release
+    project-type: pipeline
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/rcbops/rpc-gating
+            branches:
+              - "${RPC_GATING_BRANCH}"
+            credentials-id: "github_account_rpc_jenkins_svc"
+      script-path: job_dsl/release.groovy
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+      - text:
+          name: COMMAND
+          default: |
+            # the following parameters must be replaced with your values:
+            # REPO - the repo containing the project to be release
+            # SCRIPT - a script to execute to generate release notes
+            # VERSION - the version to release
+            # PREVIOUS_VERSION - Used as base version for generating release notes
+            # NOTIFICATION_ADDRESS - where to send release notifications
+            python rpc-gating/scripts/release.py \
+                --debug \
+                --org rcbops \
+                --repo REPO \
+                clone \
+                  --ref master-rc \
```

Comment:
Nit, but is the indentation here off?

Reply:
No, because ref is an arg for clone, but org and repo are top level args.